server/service/service_osquery.go

```diff
@@ -277,9 +277,12 @@ var detailQueries = map[string]struct {
 	IngestFunc func(logger log.Logger, host *kolide.Host, rows []map[string]string) error
 }{
 	"network_interface": {
-		Query: `select * from interface_details id join interface_addresses ia
-                        on ia.interface = id.interface where broadcast != ""
-                        order by (ibytes + obytes) desc`,
+		Query: `select ia.interface, address, mask, broadcast, point_to_point,
+                               id.interface, mac, id.type, mtu, metric, flags, ipackets, opackets, 
```

Comment:
Please remove `flags` and `collisions` as they seem to be unused and are missing from earlier versions of osquery.